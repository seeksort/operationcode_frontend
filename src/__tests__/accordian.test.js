import React from 'react';
import renderer from 'react-test-renderer';
import Accordian from './../shared/components/accordian/accordian';
import AccordianItem from './../shared/components/accordian/accordianItem/accordianItem';

it('renders correctly', () => {
  const tree = renderer.create(
    <Accordian>
      <AccordianItem>child 1</AccordianItem>
      <AccordianItem>child 2</AccordianItem>
    </Accordian>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
