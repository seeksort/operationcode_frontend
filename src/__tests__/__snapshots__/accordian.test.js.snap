// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className={undefined}
>
  <dl>
    <dt
      className={undefined}
    >
      <a
        aria-controls="accordion1"
        aria-expanded="false"
        className={undefined}
      >
        First Accordion heading
      </a>
    </dt>
    <dt
      className={undefined}
    >
      <a
        aria-controls="accordion1"
        aria-expanded="false"
        className={undefined}
      >
        First Accordion heading
      </a>
    </dt>
  </dl>
</div>
`;
